addons/main/CfgWeapons.hpp

```diff
@@ -15,4 +15,93 @@ class CfgWeapons {
             mass = PLATE_MASS;
         };
     };
+
+    class GVAR(autoInjector): GVAR(plate)  {
```

Comment:
you might want to look into magazines then. because they work exactly what you need here. however inventory management might be a bit of a hassle.

Reply:
Not sure how I'd implement this, honestly. Would rather it stay sorted to miscItems in arsenal, and I don't know how to run code on things to add a magzine or replace the item in inventory when it's put in.